app/chaturbate/cmd.go

```diff
@@ -65,30 +67,29 @@ func listRooms() string {
 	return s
 }
 
-func listHandler(w http.ResponseWriter, r *http.Request) {
+func listHandler(w http.ResponseWriter, _ *http.Request) {
 	fmt.Fprint(w, listRooms())
 }
 
-func debugHandler(w http.ResponseWriter, r *http.Request) {
-
-	chWorker.Lock()
-	tmp := chWorker.Map
-	chWorker.Unlock()
-
+func debugHandler(w http.ResponseWriter, _ *http.Request) {
 	x := []string{}
-	for k, _ := range tmp {
+	chWorker.RLock()
```

Comment:
use locking as chWorker contains pointers that can be modified in other goroutines